src/endpoint/iam/iam_rest.js

```diff
@@ -14,15 +14,22 @@ const IAM_XML_ROOT_ATTRS = Object.freeze({
 });
 
 const RPC_ERRORS_TO_IAM = Object.freeze({
```

Comment:
- Please do not delete what we already had, for example I know for example that `SIGNATURE_DOES_NOT_MATCH` is used.
- Could you stay with the convention of the RPC errors style (capital letters and with underscores)?

Reply:
Ok, got it its similer to S3 behaviour